packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1726,6 +1853,16 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
     }
   }
 
+  function onBlock(remainingExpirationTime: ExpirationTime) {
```

Comment:
Inline?

Reply:
I didn't want to inline this because it's one of the things we would eventually lift to the renderer. Along with the other stuff at the bottom of this module.